app/src/ui/keyboard-shortcut/keyboard-shortcut.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+
+interface IKeyboardShortCutProps {
+  /** Windows/Linux keyboard shortcut */
+  readonly keys: ReadonlyArray<string>
+  /** MacOS keyboard shortcut */
+  readonly darwinKeys: ReadonlyArray<string>
+}
+
+export class KeyboardShortcut extends React.Component<IKeyboardShortCutProps> {
+  public render() {
+    const keys = __DARWIN__ ? this.props.darwinKeys : this.props.keys
+
+    return keys.map((k, i) => {
+      return (
+        <>
+          <kbd key={k + i}>{k}</kbd>
+          {!__DARWIN__ && i < keys.length - 1 ? <> + </> : null}
```

Comment:
Sorry, my bad, I discussed it with the team and prefer it without the spaces around. Other than that is looking great!!!
```suggestion
          {!__DARWIN__ && i < keys.length - 1 ? <>+</> : null}
```

Reply:
I made the change myself from the suggestion. I will test this tomorrow and if everything is ok I will go ahead and merge it 😄